code/style.css

```diff
@@ -1,4 +1,349 @@
-html {
-  background: #0025ff;
+/**** over all styling ****/
+body{
+ background-color: #f0f0f0;
+ color: #354649;
+ margin: 0 auto;
+ padding: 0;
+ padding-top: 70px;
+ width: 100%;
+ font-family: "Cardo", serif;
+}
+
+a {
+  text-decoration: none;
+  color: #354649;
+}
+
+/****styling of header****/
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #A3C6C4;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  position: fixed;
+  top: 0;
+}
+
+.icon {
+  margin-left: 20px;
+  font-size: 48px;
+  color: #354649;;
+}
+
+.hamburger {
+  display: flex;
+  height: 24px;
+  width: 24px;
+  margin: 0 10px;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.hamburger div {
+  width: 24px;
+  height: 2px;
+  margin-right: 30px;
+  background: #354649; 
+} 
+
+h1 {
+  font-size: 24px;
+  margin-left: 20px;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  text-decoration: none;
+  font-size: 18px;
+}
+
+nav a:hover {
+  font-weight: bold;
+}
+
+/**** styling of big news section ****/
+
+.big-news {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.big-news-image {
+  margin: 20px;
+  width: 90%;
+}
+
+.big-news:hover {
+  background-color: #fafafa;
+}
+
+h2 {
+  margin-top: 10px;
+  font-size: 24px;
+}
+
+.big-news p {
+  font-size: 20px;
+  margin-bottom: 30px;
+}
+
+.news-text {
+  text-align: center;
+  margin: 0 20px 30px 20px;
+}
+
+.news-text a:hover {
+  font-weight: bold;
+}
+
+.read-link {
+  background-color: #A3C6C4;
   color: #fff;
+  font-size: 20px;
+  padding: 2px 8px;
+  border-radius: 20px;
+}
+
+/**** styling of card container ****/
+
+h3 {
+  font-size: 20px;
+  margin: 20px 0 -10px;
+}
+.card-container p {
+  font-size: 18px;
+}
+
+.card-container {
+  display:flex;
+  flex-direction: column;
+  text-align: center; /* for the h3 & p*/
+  align-items: stretch;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  
+}
+
+.cards a {
+  background: #fff;
+  width: 90%;
+  box-shadow: 5px 5px 0 0 #E0E7E9;
+  margin: 10px 5px;
+}
+
+.card-image {
+  margin: 20px 5px;
+  width: 90%;
+}
+
+.card-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 10px;
+}
+
+.cards a:hover {
+  background-color: #fafafa;
+  border: solid 1px #E0E7E9;
+}
+
+h4 {
+  margin: 0px;
+}
+
+/**** styling of footer ****/
+
+footer {
+  background-color: #A3C6C4;
+  margin: 0;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+h5 {
+  font-size: 16px;
+  font-weight: bolder;
+  margin-bottom: -10px;
```

Comment:
Almost always there is better a solution than margin with values lower than 0. Perhaps some lower padding or margin (but still higher than 0) somewhere else?